lib/puppet/ssl/validator.rb

```diff
@@ -24,7 +24,42 @@ def self.no_validator()
   # @api public
   #
   def self.default_validator()
-    Puppet::SSL::Validator::DefaultValidator.new()
+    Puppet.deprecation_warning("Puppet::SSL::Validator.default_validator is deprecated. Use Puppet::SSL::Validator.cert_auth_validator if the host has a signed cert, or Puppet::SSL::Validator.best_validator if the host certificate signing status is unknown.")
```

Comment:
Thanks for catching this; it turns out that this logging was breaking the appveyor builds as well. I've fixed it.